### PR TITLE
Update/pt/readme

### PR DESCRIPTION
**Pull request by jralmaraz**
Update README-pt.md

### CODE REVIEW

**Comment by jailton** [comment]
Hi, @jralmaraz! You need to remove commit 8fcedaaa from this PR.

File: README-pt.md [hunk 1]
```diff
@@ -144,6 +144,15 @@ sudo launchctl load -w /Library/LaunchDaemons/limit.maxfiles.plist
...
 Esta solução funciona tanto para o MacOS Catalina quanto para o MacOS Mojave.
 
+### Troubleshooting for Out of Memory
+
+Se você executar o comando `make container-serve` e retornar o seguinte erro:
+```
+make: *** [container-serve] Error 137
+```
+
+Verifique a quantidade de memória disponível para o executor/runtime de container. No caso de Docker Desktop para macOS, abra o menu "Preferences..." -> "Resources..." e tente disponibilizar mais memória.
```
Comment (jailton) [inline README-pt.md:154]: Two corrections:
1. From: "executor/runtime de container"; To: "agente de execução de contêiner"
1. From: "No caso **de** Docker"; To: "No caso **do** Docker"

**Review by jailton — DISMISSED** [review]

File: README-pt.md (hunk 1 above)
Comment (jralmaraz) [reply README-pt.md]: Thanks @jailton ! Changes done, let me know if there's something pending. Cheers!

File: README-pt.md [hunk 2]
```diff
@@ -144,6 +144,15 @@ sudo launchctl load -w /Library/LaunchDaemons/limit.maxfiles.plist
 
 Esta solução funciona tanto para o MacOS Catalina quanto para o MacOS Mojave.
 
+### Troubleshooting for Out of Memory
```
Comment (rikatz) [inline README-pt.md:147]: s/Troubleshooting for/Troubleshooting para erro de "Out of Memory"

**Comment by rikatz** [comment]
One small change and we're good to go :)

**Review by yagonobre — COMMENTED** [review]
After you fix @rikatz comments, SGTM.

Thanks!.

**Comment by jralmaraz** [comment]
Thanks @rikatz  and @yagonobre. Pushed 4062bec . Cheers!

**Review by yagonobre — APPROVED** [review]
/lgtm

**Review by rikatz — APPROVED** [review]

**Review by 0sajadi — COMMENTED** [review]
Gotcha

**Pull request merged**